components/layout/index.jsx

```diff
@@ -1,20 +1,36 @@
-import React from 'react'
+import React, { useState } from 'react'
+import classNames from 'classnames'
 import Header from '../header'
 import Head from '../head'
 import './index.css'
 import Sidebar from '../sidebar'
 
 const Layout = ({ children }) => {
+  const [isSidebarVisible, toggleSidebarVisibility] = useState(false)
+
+  const handleCLick = e => {
+    const { name, value } = e.target
+    if (['sidebar.show', 'sidebar.close'].includes(`${name}.${value}`))
+      toggleSidebarVisibility(!isSidebarVisible)
+  }
+
+  // TODO: Ask Viktor how aria should be handled
```

Comment:
@viktor-yakubiv Could you please help me out here? DIV element bellow has onClick listener and I don't know which aria role should be used. 😕

Reply:
Toggle button should have `aria-haspopup="true" aria-controls="sidebar-id" aria-expanded="false"`. The last one should change when sidebar toggles.

The most tricky one is ID in the current application structure. That is why I would do layout in a single file somehow. Otherwise you have to pass this ID to header and then pass it to the bottom.

Reply:
Thanks, unfortunately, there were other a11-js warnings. I left there TODO.